gammapy/cube/tests/test_simulate.py

```diff
@@ -58,3 +61,56 @@ def test_simulate():
     )
     assert_allclose(dataset.psf.data[5, 32, 32], 0.04203219)
     assert_allclose(dataset.edisp.data.data[10, 10], 0.85944298, rtol=1e-5)
+
+
+def dataset_maker():
+    position = SkyCoord(0.0, 0.0, frame="galactic", unit="deg")
+    energy_axis = MapAxis.from_bounds(
+        1, 100, nbin=30, unit="TeV", name="energy", interp="log"
+    )
+
+    exposure = Map.create(
+        binsz=0.02,
+        map_type="wcs",
+        skydir=position,
+        width="2 deg",
+        axes=[energy_axis],
+        coordsys="GAL",
+        unit="cm2 s",
+    )
+
+    spatial_model = GaussianSpatialModel(
+        lon_0="0 deg", lat_0="0 deg", sigma="0.2 deg", frame="galactic"
+    )
+
+    spectral_model = PowerLawSpectralModel(amplitude="1e-11 cm-2 s-1 TeV-1")
+    skymodel = SkyModel(spatial_model=spatial_model, spectral_model=spectral_model)
+
+    geom = WcsGeom.create(
+        skydir=position, binsz=0.02, width="5 deg", coordsys="GAL", axes=[energy_axis]
+    )
+
+    t_ref = Time("2010-01-01T00:00:00")
+    t_min = 0 * u.s
+    t_max = 30000 * u.s
+
+    gti = GTI.create(start=t_min, stop=t_max)
+
+    dataset = get_map_dataset(
+        sky_model=skymodel, geom=geom, geom_etrue=geom, edisp=True
+    )
+    dataset.gti = gti
+
+    return dataset
+
+
```

Comment:
I think this test requires a `@requires_data()` decorator. In Gammapy we have to mark tests, which rely on test data in `$GAMMAPY_DATA` (the cat IRFs in this case...). You can import the `@requires_data()` decorator from `from gammapy.utils.tests import requires_data`